### PR TITLE
Moved scrip in scrip.js, trigger datetimepicker on class

### PR DESCRIPTION
**Pull request by livike**
as you described in the Issue
pls sync your master first :)

### CODE REVIEW

File: static/js/script.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+	jQuery(document).ready(function($){
+			$(".datetime-widget").datetimepicker({ format:'Y-m-d H:i'	});
+		});
```
Comment (jurecuhalev) [inline static/js/script.js]: Missing newline, and you don't need tab before jQuery
There's whitespace after `format:`

Comment (livike) [inline static/js/script.js]: fixed

File: static/js/script.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+jQuery(document).ready(function($){
+	$(".datetime-widget").datetimepicker({ format: 'Y-m-d H:i' });
+});
```
Comment (jurecuhalev) [inline static/js/script.js]: missing newline

File: web/templates/base.html [hunk 3]
```diff
@@ -89,5 +89,6 @@
 		<script type="text/javascript" src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 		<script type="text/javascript" src="{% static 'js/countdown.js' %}"></script>
 		{% block custom_js %}{% endblock custom_js %}
+		<script type="text/javascript" src="{% static 'js/script.js' %}"></script>
```
Comment (jurecuhalev) [inline web/templates/base.html:91]: custom_js should come after script.js (so you can override it from child template)

Comment (livike) [inline web/templates/base.html:91]: In this case custom_js have the script link to the datetimepicker.js. I thanked, that you can not reference something in script.js, that will be loaded after...

Comment (jurecuhalev) [inline web/templates/base.html:91]: That is true, you should move datepicker to base.html then too :)
(and  we can discuss how to pack javascript files next time)

**Pull request closed without merging**